### DISCUSSION TITLE
Using Viessmann ZK03840 with ViCare AND zigbee2mqtt

### DISCUSSION THREAD

**Discussion by smhex** [Koenkk/zigbee2mqtt#10392] [Devices]
Hi, I am new to Zigbee and I was wondering if I can use the Viessmann valve with zigbee2mqtt AND ViCare in parallel? As I have read a pairing is required and I am afraid of getting the valve removed from ViCare once I have a paired it with another Zigbee adapter. Can you confirm this?

Many thanks
smhex

**Comment by sjorge** [comment]
The TRV can only be paired to one of them at once. So you lose it in the ViCare hub if you pair it with z2m and visa versa

**Reply by smhex** [reply]
Thank you! I was hoping to get the values via zigbee. Viessmann does not publish them via their API...

**Reply by p2phuture** [reply]
Hello did you get any further with this? I bought a Zigbee repeater hoping the network would automatically extend (as it would with a wifi repeater) but this does not seem to have worked. Would you have any advice?

Thank you

**Reply by smhex** [reply]
No, I did not find a solution for this issue.
